docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+version: "3"
+
+volumes:
+  gnonode: {}
+  faucet: {}
+
+services:
+  gnoland:
+    image: ghcr.io/gnolang/gno/gnochess:latest
+    environment:
+      - LOG_LEVEL=4
+    command: ["gnoland", "start", "-genesis-max-vm-cycles", "100000000"]
+    volumes:
+      - "gnonode:/opt/gno/src/testdir"
+      - "./util/devnet/config.toml:/opt/gno/src/gno.land/testdir/config/config.toml"
+    working_dir: /opt/gno/src/gno.land
+    ports:
+      - 26657:26657
+    restart: on-failure
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "100m"
+
+  deployer:
+    build: ./util/devnet/deployer
+    restart: on-failure
+    depends_on:
+      - gnoland
+    volumes:
+      - "./realm:/realm"
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "100m"
+
+  web:
+    build: ./web
+    ports:
+      - 1313:1313
+    restart: on-failure
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "10"
+        max-size: "100m"
+    volumes:
+      - ./web:/app
+    environment:
+      VITE_GNO_WS_URL: "http://localhost:26657/websocket"
```

Comment:
Please add the `VITE_GNO_JSONRPC_URL` tag 🙏 

However, we will not use it if PR #135 is confirmed as resolving the timeout issue